ui/imports/shared/views/PasswordView.qml

```diff
@@ -78,14 +76,26 @@ ColumnLayout {
         property bool containsNumbers: false
         property bool containsSymbols: false
 
-        readonly property var validator: RegExpValidator { regExp: /^[!-~]{0,64}$/ } // That incudes NOT extended ASCII printable characters less space and a maximum of 64 characters allowed
+        readonly property var validatorRegexp: /^[!-~]{0,64}$/
+        readonly property var validator: RegExpValidator { regExp: d.validatorRegexp } // That incudes NOT extended ASCII printable characters less space and a maximum of 64 characters allowed
+        readonly property string validatorErrMessage: qsTr("Only letters, numbers, underscores and hyphens allowed")
```

Comment:
But we do allow spaces too, right? So this error message is a bit confusing

Reply:
The input field should accept space, but invalidate the password to let the user know space is not allowed.

Reply:
Sorry, forgot to link it to the bug. There's more info in there.
> Closing: https://github.com/status-im/status-desktop/issues/8385

Reply:
Ah I see!